datasetcard.md

```diff
@@ -69,15 +69,15 @@ dataset_info:
     #       dtype: string
     #     - name: answer_start
     #       dtype: int32
-  config_name: {config_name}  # Example for glue: sst2
+  config_name: {config_name}  # Name of the configuration (or subset). Example for glue: sst2
   splits:
     - name: {split_name_0}                  # Example: train
       num_bytes: {split_num_bytes_0}        # Example for SQuAD: 79317110
       num_examples: {split_num_examples_0}  # Example for SQuAD: 87599
   download_size: {dataset_download_size}   # Example for SQuAD: 35142551
   dataset_size: {dataset_size}             # Example for SQuAD: 89789763
 
-# It can also be a list of multiple configurations:
+# It can also be a list of multiple configurations (or subsets):
```

Comment:
here i think it's fine this way

Reply:
or this:
> subsets (also called "configurations")

Reply:
https://github.com/huggingface/hub-docs/pull/1348